### PR TITLE
WL-2754 Rename 'My Active Sites' to 'My Sites'

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: user-tool-prefs/tool/src/bundle/user-tool-prefs_en_GB.properties [hunk 1]
```diff
@@ -7,3 +7,9 @@ privacy_stmt_show=." Others can see your name in the site members list and list
 privacy_stmt_hide=." Others cannot see your name in the site members list or list of users present.
 privacy_show_all_set=Your status in all sites is now set to visible.  Others can see your name in the site members list and list of users present.
 privacy_hide_all_set=Your status in all sites is now set to hidden.  Others cannot see your name in the site members list or list of users present.
+
+# WL-2754 rename My Active Sites to My Sites
```
Comment (buckett) [inline user-tool-prefs/tool/src/bundle/user-tool-prefs_en_GB.properties:11]: You added the props but haven't dropped Active.

Comment (ghost) [inline user-tool-prefs/tool/src/bundle/user-tool-prefs_en_GB.properties:11]: Useless!

**Comment by ghost** [comment]
Fixed in 57611d2

File: user-tool-prefs/tool/src/bundle/user-tool-prefs_en_GB.properties [hunk 2]
```diff
@@ -7,3 +7,9 @@ privacy_stmt_show=." Others can see your name in the site members list and list
...
+tab_inst_3_alt=My Sites
+tab_inst_4_alt=This whole list will appear in your My Active Sites tab.
```
Comment (buckett) [inline user-tool-prefs/tool/src/bundle/user-tool-prefs_en_GB.properties]: Taking it carefully? Another reference to Active here.

Comment (ghost) [inline user-tool-prefs/tool/src/bundle/user-tool-prefs_en_GB.properties]: Not my day today clearly!

**Pull request merged**